test/vault/unit/stanley/Stanley-SetStrategy.ts

```diff
@@ -104,6 +105,42 @@ describe("Stanley -> StrategyChanged", () => {
             ).to.be.true;
         });
 
+        it("Should setup AAVE strategy when balance on strategy is zero", async () => {
+            //given
+            const NewStrategyAave = await hre.ethers.getContractFactory("MockStrategy");
+            const newStrategyAave = await NewStrategyAave.deploy();
+            await newStrategyAave.setShareToken(aDAI.address);
+
+            await newStrategyAave.setAsset(DAI.address);
+            const oldStrategyAddress = strategyAave.address;
+
+            const oldStrategyBalanceBefore = await strategyAave.balanceOf();
+            const newStrategyBalanceBefore = await newStrategyAave.balanceOf();
+
+            //when
+            await expect(stanley.setStrategyAave(newStrategyAave.address))
```

Comment:
Please extract stanley.setStrategyAave(newStrategyAave.address) to named constant. I don't know what is returning by setStrategyAave. In general, it is strange that the set is returning something.

Reply:
This doesn't return anything, 
this is construction to catch events in tests with hardhat